src/components/BaseModal/BaseModal.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import './BaseModal.css';
+import computeClassNames from '../../utils/compute-class-names';
+
+interface Props {
+  isClosed?: boolean;
+  renderHeader?: () => React.ReactNode;
+  onClose?: (value: boolean) => void;
+  children?: React.ReactNode;
+  className?: string;
+}
+
+function BaseModal({ isClosed = false, renderHeader, onClose, children, className = '' }: Props) {
+  const computedClassNames = computeClassNames({
+    'base-modal-container--open': isClosed,
+    'base-modal-container--closed': !isClosed
+  });
+
+  return (
+    <div className={`base-modal-container ${computedClassNames} ${className}`}>
```

Comment:
Si el modal esta cerrado es decir, si `isOpen` es igual a `false` seria ideal que el modal no se renderizara en el DOM
```suggestion
    { isOpen ? <div className={`base-modal-container ${computedClassNames} ${className}`}> : null }
```